src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx

```diff
@@ -13,117 +14,117 @@ redirects:
 
-<Callout variant="tip">
-  If you'd rather not compile the plugin yourself, you can download pre-compiled versions from our [GitHub repository's releases page](https://github.com/newrelic/newrelic-fluent-bit-output/releases).
-</Callout>
+You can install it manually using Helm, although this option is not recommended. Run this command to install the repo:
 
-## Upgrade the Fluent Bit plugin [#upgrade-plugin]
+```bash
+    helm repo add newrelic https://helm-charts.newrelic.com
+```
 
-Before you upgrade your Fluent Bit plugin, run the following NRQL query to find the current versions of the output plugin being used in your system:
+To update the repo, run this command:
 
-```sql
-    FROM K8sContainerSample 
-    SELECT latest(containerImage) 
-    WHERE podName like '%newrelic-logging%' 
-    FACET clusterName
+```bash
+    helm repo update newrelic
 ```
 
-<Callout variant="caution">
-  Fluent Bit output plugin versions 1.16.0 through 1.19.2 are affected by a [security vulnerability (CVE-2024-4323)](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-4323). If you're using one of these versions, upgrade to version 2.0.0 or higher. For more information on this, see our [security bulletin NR24-01 - Fluent Bit](/docs/security/new-relic-security/security-bulletins/security-bulletin-nr24-01/).
+<Callout variant="tip">
+  See our [supported configuration parameters](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging#supported-configuration-parameters) in our `newrelic-logging` Helm chart.
 </Callout>
 
-To upgrade, follow the [installation instructions](#fluentbit-plugin) or grab the latest pre-compiled version from our [GitHub repository](https://github.com/newrelic/newrelic-fluent-bit-output/releases).
+See [Uninstall Kubernetes integration](/docs/kubernetes-pixie/kubernetes-integration/installation/uninstall-kubernetes/) if you want to uninstall it.
 
-## Install Fluent Bit output plugin [#fluent-bit]
+### Using a Docker image [#helm-docker-image]
 
-New Relic has a [Fluent Bit](https://fluentbit.io/) [output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
+Once you've published the container with the plugin installed, it serves as a base image for our Kubernetes integration. If you've your own custom kubernetes integration, we recommend using our [Docker image](https://hub.docker.com/r/newrelic/newrelic-fluentbit-output) that comes with the [`newrelic-fluent-bit-output`](https://github.com/newrelic/newrelic-fluent-bit-output) plugin. Or you can use the Docker image as a base image and layer your own custom configuration files.
```

Comment:
@nbaenam it seems like the first sentence and the last are saying the same thing, right? Could you just drop the first sentence: Once you've published the container with the the plugin installed, it serves as a base image for our Kubernetes integration.

Reply:
Done